Sources/SwiftJWT/Claims.swift

```diff
@@ -137,6 +137,9 @@ public enum ClaimKeys: String {
 
     /// Given name(s) or first name(s) (OpenID)
     case given_name
+    
+    /// The token subject’s group memberships that will be mapped to Java EE style application level roles in the MicroProfile service container (MicroProfile)
+    case groups
```

Comment:
Do we need a comment at the top of ClaimKeys to indicate where the MicroProfile claims are defined? (similar to the README statement).  There's a comment for OpenID, for example.

Reply:
I've done it in the latest commit 😄